frontend/packages/kubevirt-plugin/src/k8s/helpers/vm-clone.ts

```diff
@@ -103,7 +103,7 @@ export class VMClone {
 
         if (pvc) {
           const clonedDVTemplate = new DataVolumeTemplate({
-            name: joinIDs(name, pvcName, 'clone'),
+            name: joinIDs(name, pvcName, getRandomChars(5)),
```

Comment:
if we want to be consistent we should use volume/disk name

Reply:
Done!